src/libsystemd/sd-device/sd-device.c

```diff
@@ -2150,7 +2150,7 @@ _public_ int sd_device_get_sysattr_value(sd_device *device, const char *sysattr,
                         return r;
 
                 /* drop trailing newlines */
-                while (size > 0 && strchr(NEWLINE, value[--size]))
+                while (size > 0 && value[--size] == NEWLINE)
```

Comment:
NEWLINE is a string...

Reply:
Hah, yeah, I was stupid. tbh. I thought I had not opened the PR yet when I noticed …